contrib/read_instrumentation.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
```

Comment:
Please add some documentation here describing that the script has to be used in conjunction with the above configuration option. This feature has to be better documented.